packages/dds/tree/src/test/shared-tree/fuzz/fuzzEditGenerators.ts

```diff
@@ -542,18 +504,18 @@ export const makeUndoRedoEditGenerator = (
 	const undo: UndoOp = { type: "undo" };
 	const redo: RedoOp = { type: "redo" };
 
-	const undoRedoType = createWeightedGenerator<FuzzUndoRedoType, FuzzTestState>([
+	const undoRedoType = createWeightedGenerator<UndoRedo["operation"], FuzzTestState>([
 		[undo, passedOpWeights.undo],
 		[redo, passedOpWeights.redo],
 	]);
 
 	return (state) => {
-		const contents = undoRedoType(state);
-		return contents === done
+		const operation = undoRedoType(state);
```

Comment:
nit: this doesn't really meaningfully use the output of the nested generator.

It would be a fair bit more ergonomic to do

```typescript
return createWeightedGenerator<UndoRedo, FuzzTestState>([
	[{ type: "undoRedo", operation: { type: "undo" }}, passedOpWeights.undo],
	[{ type: "undoRedo", operation: { type: "redo "}}, passedOpWeights.redo]
])
```


Also, I think "passedOpWeights" is meant to refer to the argument, so maybe rename `opWeights -> opWeightsArg` and on L500, `const opWeights = { ...defaultEditGeneratorOpWeights, ...opWeightsArg }`